04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,102 @@
 #write your code here
+
+def translate input
+    vowels = ["a", "e", "i", "o", "u"]
+
+    # determine if input is multi-words or single word
+
+    # split multi-word input into array
+    if input.include?(" ")
+        arr = input.split(" ")
+        index = 0
+        pig_arr = []
+
+        # loop through array to convert each word into pig latin
+        while index < arr.length
+            # handle words starting with vowels
+            if vowels.include?(arr[index][0])
```

Comment:
While maintaining an `index` variable does _work_, you may want to take a look at [Array#each](https://ruby-doc.org/core-3.1.2/Array.html#method-i-each), which could make loops easier to write and read.

```suggestion
        arr.each do |word|
            # handle words starting with vowels
            if vowels.include?(word[0])
```